lib/features/sks-menu/presentation/widgets/sks_menu_tiles.dart

```diff
@@ -74,3 +75,41 @@ class SksMenuDishDetailsTile extends StatelessWidget {
     );
   }
 }
+
+class SksMenuTilesLoading extends StatelessWidget {
```

Comment:
This class could be private and moved to `sks_menu_view_loading`

Reply:
There was a problem accessing context.color Theme.white Soap when SksMenuTilesLoading was moved sks_menu_view_loading. How can I fix this?

Reply:
White soap here is optional I guess (since we re using shimmer) , you can use whatever colour you prefer or use Colour consts (see my commit with sks menu or draft commit with chart)

Reply:
100% certain. Colour here could be even pink, it completely doesnt matter